lib/cucumber-api/steps.rb

```diff
@@ -106,7 +125,6 @@
   @response = CucumberApi::Response.create response
   @headers = nil
   @body = nil
-  @grabbed = nil
```

Comment:
Since we never clear this anymore, I assume instance variables get flushed automatically when we finish a scenario?